environments/icds-cas/inventory/commcare.csv

```diff
@@ -79,3 +79,21 @@ web20,100.71.188.33,webworkers,mobile_webworkers,,,MUMGCCWCDPRDW020
 web21,100.71.188.68,webworkers,mobile_webworkers,,,MUMGCCWCDPRDW021
 web22,100.71.188.62,webworkers,mobile_webworkers,,,MUMGCCWCDPRDW022
 web23,100.71.188.74,webworkers,mobile_webworkers,,,MUMGCCWCDPRDW023
+web24,100.71.188.203,webworkers,celery,shared_dir_host,hq_webworkers,MUMGCCWCDPRDWV000
```

Comment:
Kind of annoyed these hostnames won't match with the given ones from TCL, but also think its better to not re-use the same hostname on a new host